operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -58,23 +60,29 @@ func CreateOrUpdateLokiStack(
 		gwImg = manifests.DefaultLokiStackGatewayImage
 	}
 
+	openShiftVersion, isOpenShift, err := openshift.FetchVersion(ctx, k)
+	if err != nil {
+		return nil, err
+	}
+	featureGate := manifests.NewFeatureGate(isOpenShift, openShiftVersion)
```

Comment:
I think we need to wrap this in a guard using `fg.OpenShift.Enabled` so that the community-edition does not try to request `ClusterVersion`?


```suggestion
	featureGate := manifests.NewFeatureGate(false, nil)
	if fg.OpenShift.Enabled {
		openShiftVersion, isOpenShift, err := openshift.FetchVersion(ctx, k)
		if err != nil {
			return nil, err
		}
		featureGate = manifests.NewFeatureGate(isOpenShift, openShiftVersion)
	}
```

Reply:
Added a guard to FetchVersion for this, let me know what you think